packages/aws-cdk/lib/init.ts

```diff
@@ -243,31 +244,36 @@ async function initializeGitRepository() {
   }
 }
 
-async function postInstall(language: string, canUseNetwork: boolean) {
+async function postInstall(language: string, canUseNetwork: boolean, generateOnly: boolean) {
```

Comment:
Shouldn't we just not execute `postInstall` if `generateOnly` is true?

Reply:
This would also exclude `postInstallPython` (`await execute(python, '-m venv', '.env')`), is that intended?

Reply:
Yes. If we are in “generate only” mode, the creating a virtual environment is not part of the deal.